src/velocity.py

```diff
@@ -0,0 +1,218 @@
+    * lat: float. Latitude.
+    * long: float. Longitude.
+    * grid_array: array. An (M, N, K)-dimensional array containing the K-dim
+        values of the magnitude. Each MxN matrix
+        element represents the value for the k_th dimension of that magnitude in
+        the m_th, n_th latitude, longitude point on the discrete space.
+    * array_latitudes: array. An (M)-dimensional vector with the latitude
+        values corresponding to the discrete points of the grid_array.
+    * array_longitudes: array. An (N)-dimensional vector with the longitude
+        values corresponding to the discrete points of the grid_array.
+
+    Returns:
+    ------
+    * velocity. array. A vector with the composed sum of velocities, with the
+        latitude and longitude components on that order.
+    """
+    interpolated_stream_velocities = interpolate_discrete(lat, long,
+                                                          stream_velocities,
+                                                          stream_velocities_latitudes,
+                                                          stream_velocities_longitudes)
+    velocity = boat_velocity + interpolated_stream_velocities
+
+    return velocity
+
+
+def boat_movement(boat_velocity, lat, long, stream_velocities,
+                  stream_velocities_latitudes, stream_velocities_longitudes,
+                  ts):
+    """
+    Computes the new position of the boat given the boat velocity, the steams velocity and the timestamp. Timestamp
+    is assumed to be so small that the stream velocity does not change from the initial point to the final one.
+
+    Params:
+    ------
+    boat_velocity: array. Vector containing the latitude and longitude components of the boat velocity, in that order.
+    lat: float. Latitude.
+    long: float. Longitude.
+    grid_array: array. An (M, N, K)-dimensional array containing the K-dim values of the magnitude. Each MxN matrix
+        element represents the value for the k_th dimension of that magnitude in the m_th, n_th latitude, longitude point
+        on the discrete space.
+    array_latitudes: array. An (M)-dimensional vector with the latitude values corresponding to the discrete points of
+        the grid_array.
+    array_longitudes: array. An (N)-dimensional vector with the longitude values corresponding to the discrete points of
+        the grid_array.
+    ts: float. The time increment interval [THIS VALUE IS ASSUMED TO BE SMALL SO THE DISPLACEMENT IS SHORT].
+
+    Returns:
+    ------
+    new_latitude, new_longitude. floats. The values for the latitude and longitude new positions.
+
+    """
+
+    velocity = velocity_composition(boat_velocity, lat, long, stream_velocities,
+                                    stream_velocities_latitudes,
+                                    stream_velocities_longitudes)
+    new_latitude = lat + velocity[0] * ts
+    new_longitude = lat + velocity[1] * ts
```

Comment:
Error. Debería ser long. 

```suggestion
    new_longitude = long + velocity[1] * ts
```